specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/customRecommedations.json

```diff
@@ -0,0 +1,384 @@
+            "$ref": "./examples/CustomRecommendations/PutByManagementGroupCustomRecommendation_example.json"
+          }
+        },
+        "tags": [
+          "CustomRecommendations"
+        ],
+        "description": "Creates or updates a custom recommendation over a given scope",
+        "operationId": "CustomRecommendations_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/customRecommendationName"
+          },
+          {
+            "$ref": "#/parameters/customRecommendationBody"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/CustomRecommendation"
+            }
+          },
+          "200": {
+            "description": "OK - Updated",
+            "schema": {
+              "$ref": "#/definitions/CustomRecommendation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v2/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete a custom recommendation over subscription scope": {
+            "$ref": "./examples/CustomRecommendations/DeleteBySubscriptionCustomRecommendation_example.json"
+          },
+          "Delete a custom recommendation over security connector scope": {
+            "$ref": "./examples/CustomRecommendations/DeleteBySecurityConnectorCustomRecommendation_example.json"
+          },
+          "Delete a custom recommendation over management group scope": {
+            "$ref": "./examples/CustomRecommendations/DeleteByManagementGroupCustomRecommendation_example.json"
+          }
+        },
+        "tags": [
+          "CustomRecommendations"
+        ],
+        "description": "Delete a custom recommendation over a given scope",
+        "operationId": "CustomRecommendations_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/customRecommendationName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - custom recommendation was deleted"
+          },
+          "204": {
+            "description": "No Content - custom recommendation does not exist"
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/types.json#/definitions/CloudError"
```

Comment:
>"../../../common/v2/types.json#/definitions/CloudError" [](http://example.com/codeflow?start=22&length=55)

please always consider use common type in
../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse